### DISCUSSION TITLE
Fuel Cell Model

### DISCUSSION THREAD

**Discussion by MrKrabb** [DLR-VSDC/IEEE-MVC-2023#4] [Q&A] [answered]
I do not quite understand the Fuel Cell Model. First of all, the unit of the H2 consumption is curious to me because our objective is to tranform the chemical energy of H2 into electrical energy that would be 33kWh/kg for 100% efficiency. With the unit (kg/kW) I think the efficiency is also meant, which would be (kg/kWh). In source 27 the unit is the also kg/kW but I think it is also wrong too. This can by shown calculating the assumed energy of one kg of H2 which is for example at P=20kW: 1/(0,05kg/kWh*0,6)=33,3 kWh/kg which is correct. This is the reason why I think kWh/kg would be correct. The next thing is that the H2 consumption should be calculated by 1/(33kWh/kg*eff(P)). This is obviously not the case in the given tables because the minimal H2 consumption in kWh/kg is not at the same power as the maximum efficiency as illustrated in Figure 6.

**Comment by JBRDLR** [comment, marked as answer]
@MrKrabb you are right, we are aware of this issue (there is a unit confusion already in the referenced paper) and we are preparing an update model with a different parameterization of the fuel cell. At the moment we are testing the new release (beta 2) and hope we can bring it online the next days.  Here already a preliminary screenshot what will be changed:
Fuel cell consumption:
![image](https://user-images.githubusercontent.com/110536277/209023750-09991cff-62c8-4ed9-8c4c-f7ce40fff9d1.png)
In the model we introduce a missing multiplicator:
![image](https://user-images.githubusercontent.com/110536277/209023769-7aac57b9-31ec-40b0-bb32-d31e487fa8a3.png)
